### PR TITLE
Add artifact migration for better-tostring

### PR DESCRIPTION
**Pull request by kubukoz**
See https://github.com/polyvariant/better-tostring/pull/75.

I will un-draft when the version is synced to Central :)

### CODE REVIEW

File: modules/core/src/main/resources/artifact-migrations.conf [hunk 1]
```diff
@@ -208,5 +208,10 @@ changes = [
     artifactIdBefore = argonaut-refined_6.2
     artifactIdAfter = argonaut-refined_6.3
     initialVersion = 1.3.0
+  },
+  {
+    groupIdAfter = org.polyvariant
```
Comment (fthomas) [inline modules/core/src/main/resources/artifact-migrations.conf:213]: @kubukoz The `groupIdBefore` is missing. Too bad this does not result in parse / test error.

Comment (kubukoz) [reply modules/core/src/main/resources/artifact-migrations.conf:213]: Thanks, I'll fix it later (need to figure out why publishing doesn't work first)

**Comment by kubukoz** [comment]
Please review @fthomas :)

**Review by exoego — APPROVED** [review]
LGTM!

**Pull request merged**